.gitlab/nersc-gitlab-ci.yml

```diff
@@ -0,0 +1,70 @@
+# Kokkos nightly tests at NERSC machine Perlmutter.
+stages:
+  - test
+  - clean_up
+
+default:
+  tags:
+    - perlmutter-login01
+  before_script:
+    - export SOURCE_DIR=$SCRATCH/.nightly/testing/kokkos
```

Comment:
Where will `SCRATCH` be defined?

Reply:
No, it was related to my trying to understand if the workspace is unique per build